lib/commands/details.rb

```diff
@@ -0,0 +1,64 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require_relative './google_maps_api'
+
+module Commands
+  class Details < GoogleMapsAPI
+    extend T::Sig
+
+    sig { override.returns(String) }
+    def self.name
+      'details'
+    end
+
+    sig { override.returns(String) }
+    def self.help
+      <<~HELP
+        details <place>: returns available info about a specific place listed on Google Maps.
+        If there are more than one match, the command returns a list of the top 10 results instead.
+
+        Examples:
+        details living computers museum, seattle wa
+        details korean restaurant victoria bc
+      HELP
+    end
+
+    sig { override.returns(String) }
+    def response_body
+      return api_key_missing unless api_key_exists?
+
+      return help if arg_text === ''
+
+      begin
+        places = Google::Maps.places(arg_text)
+        place = Google::Maps.place(places.first.place_id)
+      rescue StandardError => error
+        return "NO RESULTS FOR \"#{arg_text}\"" if error.message === 'Google did not return any results: ZERO_RESULTS'
+        return error.message
+      rescue
```

Comment:
If you give a `rescue` block with no error type, it defaults to `StandardError` anyways, so this block is redundant. If you're trying to capture every error ever, you need to `rescue Exception`, which is a parent class of `StandardError`.